packages/astro/test/fixtures/astro-cookies/src/pages/from.astro

```diff
@@ -0,0 +1,4 @@
+---
+console.log('cookie value: ', Astro.cookies.get('my_cookie')?.value); 
```

Comment:
Should we add another test where in the first page we set a `lorem` value, then in the rewritten page we set a `ipsum` value, and assert that the cookie has `ipsum` as value?